src/store/opportunities/slice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice, isAnyOf } from '@reduxjs/toolkit';
+
+import { IOpportunity } from 'src/common/types';
+import { loadOpportunities } from './actions';
+
+interface IOpportunityState {
+  opportunitiesLoading: boolean;
+  opportunities?: IOpportunity[];
+}
+
+const initialState: IOpportunityState = {
+  opportunitiesLoading: false,
+};
+
+const { reducer, actions } = createSlice({
+  name: 'opportunity',
+  initialState,
+  reducers: {},
+  extraReducers: builder => {
+    builder.addCase(loadOpportunities.pending, state => {
+      state.opportunitiesLoading = true;
```

Comment:
Wouldn't it be enough just to name it as "loading"?

Reply:
No, because we can also have for instance `expandedOpportunityLoading` and these are 2 different `Loading`'s